src/components/Questionnaire/ValueSetSelect.tsx

```diff
@@ -388,7 +388,8 @@ export default function ValueSetSelect({
     (system === "system-additional-instruction" ||
       system === "system-route" ||
       system === "system-body-site" ||
-      system === "system-administration-method")
+      system === "system-administration-method" ||
+      system === "system-as-needed-reason")
```

Comment:
don't you think an `[...].includes(system)` would make this more readable here? 🤔